lib/galaxy/model/__init__.py

```diff
@@ -792,6 +792,23 @@ def __init__(self, email=None, password=None, username=None):
         self.active = False
         self.username = username
 
+    def get_user_data_tables(self, data_table: str):
+        session = object_session(self)
+        metadata_select = (
+            select(HistoryDatasetAssociation)
+            .join(Dataset)
+            .join(History)
+            .filter(
```

Comment:
```suggestion
            .where(
```
Same behavior, but newer syntax. I also like it because it's less prone to be mistook for `filter_by`.